controllers/util/objectmeta.go

```diff
@@ -23,3 +25,10 @@ func ObjectMetaEmbedded(objectMeta *metav1.ObjectMeta) metav1.ObjectMeta {
 func ResourceIsDeleted(obj client.Object) bool {
 	return !obj.GetDeletionTimestamp().IsZero()
 }
+
+func ProtectedPVCNamespacedName(pvc ramen.ProtectedPVC) types.NamespacedName {
```

Comment:
Is there any reason you prefer this function specific to `ProtectedPVC` over this generic one?
```
func NamespacedName(o client.Object) types.NamespacedName {
	return types.NamespacedName{
		Namespace: o.GetName(),
		Name:      o.GetNamespace(),
	}
}
```